README.md

```diff
@@ -1,2 +1,66 @@
-# First_Project
-This Hello-world project is built using HTML and CSS.  The goal here is to master all of the tools and best practices learned about in previous steps.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> This Hello-world project is built using HTML and CSS.  The goal here is to master all of the tools and best practices learned about in previous steps.
+
+
+## Built With
+
+- HTML, CSS
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+You need to have a code editor to be able to run the commands in the terminal. VSCode is most preferred.
+
+### Setup
+You need to download the linters folder and copy it to your project.
+
+### Install
+Install the linters folder to your project.
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Marlyn_Mayienga)
+- Twitter: [@twitterhandle](https://twitter.com/Merl_Mayienga)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Marlyn_Mayienga)
+
+
```

Comment:
[OPTIONAL]: 
⚠️ This is your Hello Microverse project, and this is your third re-review. For that reason, I will be approving this PR. You did a great job using the boilerplate for the README.md 🥳  however, in the future, you MUST replace the information. Over here, it would be nice to have your GitHub handle, your twitter handle, and LinkedIn!

Reply:
Hello @melaniesigrid . In the author's section, I have my name written in the link eg: (https://github.com/Marlyn_Mayienga).
Isn't that the right way to have my name written?